src/trade/bid/bid.service.ts

```diff
@@ -59,4 +59,13 @@ export class BidService {
             where: { id_userId: { id: bidId, userId: userId } },
         })
     }
+
+    async getBidByUserId(userId: number) {
+        return this.prisma.auctionBid.findFirst({ where: { userId } })
+    }
+
+    async getHighestPrice() {
```

Comment:
Write method which takes an argument `bitPrice` and find hihhest price in `auctionBid`
if find return true else false